web/css/style.css

```diff
@@ -2,199 +2,399 @@
  * ==|== normalize ==========================================================
  */
 
-article, aside, details, figcaption, figure, footer, header, hgroup, nav, section { display: block; }
```

Comment:
please don't reformat the code of normalize.css. This would only make it harder to upgrade.

Reply:
Well, I think it's better to have one code style only per file, but ok, you're the boss, boss. ;)